gocourse03/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type (
+	Sector     string
+	rodentType string
+	FromTo     [2]Sector
+
+	Movement struct {
+		Time time.Time
+		FromTo
+		rodentId int
+	}
+
+	Rodent struct {
+		ID          int
+		Type        rodentType
+		Sector      Sector
+		UpdatedTime time.Time
```

Comment:
`Time` не варто додавати, це зайве, бо в нас же тип поля і так `time.Time`:
```suggestion
		UpdatedAt time.Time
```